markdownlint.js

```diff
@@ -161,7 +161,37 @@ function printResult(lintResult) {
     process.exitCode = 1;
   }
 
-  if (program.output) {
+  if (program.junit) {
+    const builder = require('junit-report-builder');
+    const suite = builder
+      .testSuite()
+      .name('Markdownlint')
+      .timestamp(new Date().toISOString());
+    if (results.length > 0) {
+      results.forEach(result => {
+        const {file, lineNumber, column, names, description} = result;
+        const columnText = column ? `:${column}` : '';
+        suite
+          .testCase()
+          .className(file)
+          .name(`${file}:${lineNumber}${columnText} ${names}`)
+          .failure(`${file}:${lineNumber}${columnText} ${names} ${description}`, names)
+          .time(0);
+      });
+    } else {
+      suite.testCase().className(program.args).name('Markdownlint');
+    }
+
+    suite.time(0);
+    try {
+      builder.writeTo(program.junit);
+    } catch (error) {
+      console.warn('Cannot write to output file ' + program.junit + ': ' + error.message);
+      process.exitCode = 2;
```

Comment:
Maybe update the docs or use a new exit code? https://github.com/igorshubovych/markdownlint-cli#exit-codes

Reply:
Set it to 4 and added to the readme as well.